src/EnergyPlus/Furnaces.cc

```diff
@@ -9935,6 +9950,74 @@ namespace Furnaces {
 
 	}
 
+	void
+	SetMinOATCompressor(
+		int const FurnaceNum, // index to furnace
+		std::string const FurnaceName, // name of furnace
+		std::string const cCurrentModuleObject, // type of furnace
+		std::string const CoolingCoilType, // type of cooling coil
+		std::string const CoolingCoilName, // name of cooling coil
+		std::string const HeatingCoilType, // type of heating coil
+		std::string const HeatingCoilName, // name of heating coil
```

Comment:
Well, the coils have already been read in by the time this new function was called (at end of each loop). So yes, the coil index could have been passed instead of coil name.